osgar/drivers/kloubak.py

```diff
@@ -39,6 +44,8 @@
 INDEX_FRONT_RIGHT = 0
 INDEX_REAR_LEFT = 3
 INDEX_REAR_RIGHT = 2
+INDEX_BACK_LEFT = 5
```

Comment:
the same, names are misleading

Reply:
I will rename it to: INDEX_REAR_K3_L

Reply:
Actually, we are not use the INDEX_BACK_LEFT and RIGHT in the code. I forgot resolve this thing. The solution in the update_pose function is only for K2. Should we used the third axle for encoders (K3)?